src/osd/OSD.h

```diff
@@ -796,7 +796,7 @@ class OSDService {
 
 private:
   /// throttle promotion attempts
-  atomic_t promote_probability_millis; ///< probability thousands. one word.
+  atomic_uint promote_probability_millis; ///< probability thousands. one word.
```

Comment:
i'd prefer if we were explicit about the `std::` part, especially in headers - we shouldn't assume that other headers are `using namespace std;`